siakg/keys.go

```diff
@@ -0,0 +1,172 @@
+package main
+
+// keys.go contains functions for generating and printing keys.
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	"github.com/NebulousLabs/Sia/crypto"
+	"github.com/NebulousLabs/Sia/encoding"
+	"github.com/NebulousLabs/Sia/types"
+)
+
+const (
+	// The header for all siakg files. Do not change.
+	FileHeader = "siakg"
+)
+
+// A KeyPair is the object that gets saved to disk for a signature key. All the
+// information necessary to sign a transaction is in the struct, and the struct
+// can be directly written to disk.
+type KeyPair struct {
```

Comment:
fyi you don't need to export a type to marshal it, just the fields. I guess it doesn't make much difference since this is a `main` anyway.